Gemfile.jruby-1.9.lock

```diff
@@ -723,7 +725,7 @@ DEPENDENCIES
   logstash-input-imap
   logstash-input-irc
   logstash-input-jdbc
-  logstash-input-kafka
+  logstash-input-kafka (= 3.0.0.beta4)
```

Comment:
for alpha1, @suyograo thought it would be a good idea since the 3.x version of this will make it to 5.0